tests/sklearn/test_datasets.py

```diff
@@ -275,3 +275,11 @@ def test_onehot_transformer():
     dum = pd.get_dummies(X)
     assert ohe.fit_transform(X).shape[1] == dum.shape[1] == 63
     assert dum.columns.symmetric_difference(ohe.get_feature_names_out()).empty
+
+def test_openML_store():
+    """Test to verify if the new class OpenMLStore returns the same df as before"""
+    store = OpenMLStore()
+    df_new = store.download(None, None)
+    df_old = fetch_openml(data_id=1558 if percent10 else 1461, data_home=data_home
+                      or DATA_HOME_DEFAULT, cache=cache, as_frame=True).frame
+    assert df_old == df_new
```

Comment:
Can you try `pytest test_datasets.py`? I'm confused how these two are equal. the old one is using data_id 1558 while the new one is using 1590. 

Idea is for df_new and df_old to be build with the same inputs:
```
 from aif360.sklearn.datasets.openml_datasets import DATA_HOME_DEFAULT

df_new = store.download(1461, DATA_HOME_DEFAULT)
df_old = fetch_openml(data_id=1461, data_home=DATA_HOME_DEFAULT)
```

Reply:
Done